cloudpickle/cloudpickle.py

```diff
@@ -896,6 +897,24 @@ def _gen_ellipsis():
 def _gen_not_implemented():
     return NotImplemented
 
+
+def _get_cell_contents(cell):
+    try:
+        return cell.cell_contents
+    except ValueError:
+        # sentinel used by ``_fill_function`` which will leave the cell empty
+        return _empty_cell_value
+
+
+@object.__new__
```

Comment:
I have wrapped this in a named function with a docstring.

Idk if this is better.